src/components/BaseComponents/RepoChart/useRepoChart.js

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import { getContributors } from "api";
+
+export function useContributors(contributors_url) {
+    const [contributors, setContributors] = React.useState([]);
+    React.useEffect(() => {
+        const getContributorsDetails = async () => {
+            const data = await getContributors(contributors_url);
+            setContributors(data);
+        };
+        getContributorsDetails();
+    }, []);
+
+    return {
+        contributors,
+    };
+}
```

Comment:
Don't we have a problem with data here? I bet we should add setContributors to dependencies.